### PR TITLE
feat: Create sync.yml

### PR DESCRIPTION
**Pull request by vinckr**
This should sync up oryworks repo to the other ORY repos. 
Is this all that is needed?

### CODE REVIEW

File: .github/workflows/sync.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: webfactory/ssh-agent@v0.4.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+          
+      - name: Synchronize ORY Themes
+        run: ./scripts/sync-library.sh ory/themes master Themes
```
Comment (aeneasr) [inline .github/workflows/sync.yml]: This script is missing

**Review by aeneasr — CHANGES_REQUESTED** [review]
Why would you want to copy the contents of this repository to ory/themes?

**Comment by zepatrik** [comment]
Hm guess it would be good to make sync it's own action then?

**Comment by aeneasr** [comment]
But what is it supposed to sync?

**Comment by zepatrik** [comment]
Repos from an template
Or what is the purpose of it in this repo?

**Comment by zepatrik** [comment]
*a template into a repo

**Comment by vinckr** [comment]
> Repos from an template
> Or what is the purpose of it in this repo?

"add code of conduct and cla to oryworks" automated.

with https://github.com/ory/oryworks/pull/2/commits/f035e1a93d71f8a1a2fa536a0cc56d0bb52ee481 it should sync oryworks now to the template in meta, correct? 
or do I need to change the action/workflow somewhere else?

**Comment by zepatrik** [comment]
Nope, the sync script is missing, nothing will happen. Why would you sync stuff from here to meta and not just change it in meta?

**Comment by vinckr** [comment]
Yea that was my question ^^. 
So I just have to add this here to the sync in meta, correct?

**Comment by zepatrik** [comment]
If you want to sync meta into this repo, then yes.

**Pull request closed without merging**